Aquality.Selenium.Core/src/Aquality.Selenium.Core/Elements/ElementFactory.cs

```diff
@@ -49,18 +49,15 @@ public ElementFactory(ConditionalWait conditionalWait, IElementFinder elementFin
             switch (expectedCount)
             {
                 case ElementsCount.Zero:
-                    ConditionalWait.WaitFor(driver => !driver.FindElements(locator).Any(
-                        webElement => state == ElementState.ExistsInAnyState || webElement.Displayed),
+                    ConditionalWait.WaitForTrue(() => !ElementFinder.FindElements(locator, state, TimeSpan.Zero).Any(), 
```

Comment:
should the method FindElements also be virtual?

Reply:
probably yes